javascript-forms/form-values.js

```diff
@@ -0,0 +1,11 @@
+var $contactForm = document.querySelector('form');
+var $formControls = {};
```

Comment:
You should make this object in the function, you don't want to reuse the same object every time the form is submitted.